<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7049" started-at="2018-10-31T04:27:38Z" finished-at="2018-10-31T04:27:45Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7049" started-at="2018-10-31T04:27:38Z" finished-at="2018-10-31T04:27:45Z">
      <class name="com.cashkaro.test.TestCases">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.cashkaro.test.TestCases@56235b8e]" name="setUp" is-config="true" duration-ms="5972" started-at="2018-10-31T09:57:38Z" finished-at="2018-10-31T09:57:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginApp()[pri:0, instance:com.cashkaro.test.TestCases@56235b8e]" name="loginApp" duration-ms="1046" started-at="2018-10-31T09:57:44Z" finished-at="2018-10-31T09:57:45Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.cashkaro.pages.LoginPage.loginSwithto(LoginPage.java:46)
	at com.cashkaro.test.TestCases.loginApp(TestCases.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="PASS" signature="dashBoard()[pri:2, instance:com.cashkaro.test.TestCases@56235b8e]" name="dashBoard" duration-ms="1" started-at="2018-10-31T09:57:45Z" finished-at="2018-10-31T09:57:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashBoard -->
      </class> <!-- com.cashkaro.test.TestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
